ReactWindows/ReactNative/Modules/Cursor/CursorModule.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Reflection;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+
+namespace ReactNative.Modules.Cursor
+{
+    class CursorModule : NativeModuleBase
+    {
+        public override string Name
+        {
+            get
+            {
+                return "Cursor";
+            }
+        }
+
+        /// <summary>
+        /// Set a mouse cursor
+        /// </summary>
+        /// <param name="cursor">The name of a system cursor (listed here: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corecursortype ).</param>
+        [ReactMethod]
+        public void setCursor(string cursor)
+        {
+            var convertedCursor = EnumHelpers.Parse<CoreCursorType>(cursor);
```

Comment:
>CoreCursorType [](start = 52, length = 14)

Should probably expose all `CoreCursorType` values in the `Constants` property for reference. Could do a mapping from some well-known name to the underlying int value of the enum.

Reply:
Won't this take 'Arrow' or 'Hand' and return the correct Cursor object? I'm not set up with VS 2017 yet, so I couldn't build it, but from the documentation this looks right to me.